launchflow/workflows/destroy_resource_tofu/delete_tofu_resource.py

```diff
@@ -6,17 +6,48 @@
 
 
 async def delete_tofu_resource(inputs: DestroyResourceTofuInputs):
+    # TODO: there's a bug in here we need to fix before rolling out this
+    # If the initial create fails the inputs aren't recorded and the user is unable
+    # to destroy the resource. This could happen if the user isn't logged in or something.
+    # I think the fix is to just make sure the inputs are always recorded.
     state_prefix = inputs.launchflow_uri.tf_state_prefix(module=inputs.resource.product)
+    tf_vars = inputs.resource.inputs or inputs.resource.attempted_inputs or {}
+    module_dir = inputs.resource.product
+    module_dir = f"resources/{inputs.resource.product}"
+    if inputs.resource.product == "unknown":
+        return
```

Comment:
maybe we should try to use the env's cloud provider to try deleting with an empty tf module?

Reply:
could also just leave a todo. I dont think this matters until custom resources are a thing

Reply:
I _think_ this only happens with lf cloud on initial failures so should be fine